src/theme/utils.ts

```diff
@@ -82,3 +74,13 @@ export const getColorErrors = [
     error: new PropsValidationError('No color found with that name'),
   },
 ];
+
+/**
+ * 
+ * @param figmaTokensObject The parsed objects from Figma Tokens in the src/theme/constants/ dir
+ * @returns the value of the figma token item converted into rem
+ */
+export const getFigmaTokensValue =
```

Comment:
this seems as covered but in the case of and object with `val` undefined this could fail. It is covered by `rem` function that always returns string

case: `Number({}['test'])`

In the case above if you pass it in the `rem` function you will have `NaNrem` which is a non readable variable. 

We are covered by the fact that the application will not break but UI might. Maybe we can put a test here for future use? What do you think?

Reply:
get by lodash seems pretty safe and optimal to me. maybe with a default value of 0(px).